tests/test_syslog_server.rb

```diff
@@ -34,9 +34,15 @@ def teardown
 
   def no_syslogserver
     # Turn the feature off for a clean test.
-    config('no logging server 1.2.3.4',
-           'no logging server 2003::2',
-           'no vrf context red')
+    if platform == :ios_xr
+      config('no logging 1.2.3.4',
+             'no logging 2003::2',
+             'no vrf red')
```

Comment:
Seems like a bug in IOS XR to me, but 'no vrf red' doesn't remove 'logging 2003::2 vrf red'. I had to make the following addition to get minitest passing for me:

```diff
diff --git a/tests/test_syslog_server.rb b/tests/test_syslog_server.rb
index 520f775..772bc74 100644
--- a/tests/test_syslog_server.rb
+++ b/tests/test_syslog_server.rb
@@ -36,7 +36,9 @@ class TestSyslogServer < CiscoTestCase
     # Turn the feature off for a clean test.
     if platform == :ios_xr
       config('no logging 1.2.3.4',
+             'no logging 1.2.3.4 vrf red',
              'no logging 2003::2',
+             'no logging 2003::2 vrf red',
              'no vrf red')
     else
       config('no logging server 1.2.3.4',
```